test/.eslintrc.js

```diff
@@ -0,0 +1,27 @@
+const ERROR = "error";
```

Comment:
I tried to get it working with a YAML file, and eventually ran out of patience. If you feel like contributing a PR that converts it, I'd be happy to merge it